### ISSUE TITLE
Add slots for min/max with actual and forecast values

### ISSUE THREAD

**Issue by Makin-Things** [Makin-Things/makin-things-weather-card#2]
Generated from a comment on #1 

> Add slots for min/max with actual and forecast values like
'''
Todays Min 12o (Fore 9o)
'''

I think this is a a good idea. One thing that concerns me is that with adding extra slot types and generally using the same icons for them all, without labels the numbers completely lose all context. We need to consider adding labels (maybe optional) to all fields.

**Comment by DavidFW1960** [comment]
> I think this is a a good idea. One thing that concerns me is that with adding extra slot types and generally using the same icons for them all, without labels the numbers completely lose all context. We need to consider adding labels (maybe optional) to all fields.

Well isn't this one self explanatory even if the icon is the same? Todays Min is reasonably unambiguous I think

**Comment by Makin-Things** [comment]
I was more thinking about all the existing ones that pretty much only show a temperature without any label. This one is fine.
What do you think we should call these new slots?

**Comment by DavidFW1960** [comment]
Which ones only show an icon for temp without a label? I don't see any.

> What do you think we should call these new slots?

L7 L8 and R7 R8 following the current naming...

You had previously mentioned an optional label for the custom slots so that's probably a good idea.

**Comment by Makin-Things** [comment]
Thoughts on what to use as internal names and human names for this? It is just the one slot type, or one for min and one for max?
I was thinking this is low hanging fruit and I could just knock it off, but what about when the BoM remove the Min forecast at 9am, how should it behave. I think with the way the BoM publish stuff the only time you would have both values available would be between midnight and 9am. At 9am the forecast minimum will vanish and when you think about it a forecast for something in the past makes no sense.
Basically I still think this needs more thought and will be completely different/useless if you aren't using the bom integration.

**Comment by DavidFW1960** [comment]
When they remove the min (or max) I'm ok for it to show n/a. I think it's useful to know the actual min and max and if they reached the forecast value or not. The min disappears later than 9am. It used to be 9am but is now in the afternoon I think. It's after 9 here now but still showing.

**Comment by Makin-Things** [comment]
This is done. I also added observed min/max as slot options as they weren't previously there.

**Comment by Makin-Things** [comment]
@DavidFW1960 please close if you think this is sorted, or explain what else needs work.
